lib/call.js

```diff
@@ -1862,6 +1866,42 @@ class call {
     return callmanager.options.moh
   }
 
+  /**
+  * Set the sound soup for music on ringing.
+  * @param { object } soup - sound soup as described in projectrtp
+  */
+  set mor( soup ) {
```

Comment:
Can we rethink the naming convention - this tye of thing is in standards so anyone who has telecoms experience would not understand more or moe.

In FreeSWITCH it is called https://developer.signalwire.com/freeswitch/Channel-Variables-Catalog/ringback_16353370/

I am open to another naming - but moh is generally accepted in the industry - but moe more isn't.

Perhaps something which mixes our mechanism (a soup) and ring?

i.e. ringsoup, engagedsoup?